app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    files_list = command.split(" ")
+    if len(files_list) == 3:
+        if files_list[1] != files_list[2]:
```

Comment:
And use unpacking instead of indexes:
cmd, source_path, destination_path = files_list